v12/storage-dotnet-redundancy-ha-ra-grs/Program.cs

```diff
@@ -0,0 +1,262 @@
+﻿using System.Diagnostics.Tracing;
+using System.IO;
+using Azure;
+using Azure.Core;
+using Azure.Core.Diagnostics;
+using Azure.Core.Pipeline;
+using Azure.Identity;
+using Azure.Storage;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+
+namespace StorageRedundancy
+{
+    internal class ShowRequestInfoPolicy : HttpPipelinePolicy
```

Comment:
I would move this into a separate file, to add clarity to the `Program.cs`. The `Program` might be better as a top-level program, to avoid the ceremony.